validations/src/validations.rs

```diff
@@ -265,6 +266,18 @@ pub fn validate_mint_transaction(
         }
         .into())
     } else {
+        let outputs_len = mint_tx.outputs.len();
+
+        if !mint_tx.outputs.iter().map(|vto| vto.pkh).all_equal() {
+            return Err(BlockError::DifferentPkhsInMint.into());
+        }
+
+        for output in mint_tx.outputs.iter() {
+            if outputs_len > 1 && output.value < collateral_minimum {
+                return Err(BlockError::TooSplitMint.into());
+            }
+        }
```

Comment:
These are consensus-critical statements that change the protocol in a hard-fork way. They could use code comments